cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -98,31 +100,35 @@ friendlyWithdrawals :: TxWithdrawals ViewTx era -> Aeson.Value
 friendlyWithdrawals TxWithdrawalsNone = Null
 friendlyWithdrawals (TxWithdrawals _ withdrawals) =
   array
-    [ object
-        [ "address" .= serialiseAddress addr
-        , "network" .= net
-        , "credential" .= cred
-        , "amount" .= friendlyLovelace amount
-        ]
-    | (addr@(StakeAddress net cred), amount, _) <- withdrawals
```

Comment:
What's wrong with:
```
       ...
      `, "credential" .= friendlyStakeCredential cred
       , "amount" .= friendlyLovelace amount
        ]
    | (addr@(StakeAddress net cred), amount, _) <- withdrawals
    ...
```

Reply:
Diving into address is another level of abstraction.

Reply:
Yes but you create another separate function that has to go to that level of abstraction anyways. This is less concise.

Reply:
1. That was wrong explanation. I should rather explain it with a data-structure-driven logic. We have a separate function for each node in the data structure. `friendlyStakeAddress` is the handler for `StakeAddress` type, and is used in several places (2, to be precise).
2. No network information in your version.